### ISSUE TITLE
Invisible underline in integrated terminal window

### ISSUE THREAD

**Issue by wyxd** [Microsoft/vscode#46900]
- VSCode Version: 1.21.1
- OS Version: Ubuntu 16.04

Steps to Reproduce:

1. Open the integrated terminal in vs code.
2. Type underline "_" in the window.

The underline symbol cannot be seen, but can be recognized by the terminal. 

Does this issue occur when all extensions are disabled?: Yes

**Comment by Tyriar** [comment]
Duplicate https://github.com/Microsoft/vscode/issues/35901

**Comment by certik** [comment]
There are two ways to fix this in Ubuntu (for me it happens in 17.10). 

1. Put this into your settings:
   ```json
    "window.zoomLevel": -0.001,
   ```
   That seems to fix the problem with the original vscode font.

2. Put this into your settings:
   ```json
    "terminal.integrated.fontSize": 16,
    "terminal.integrated.fontFamily": "Ubuntu mono",
   ```
   This will use the same font as in the gnome-terminal in Ubuntu, and the size will be the same as in the original vscode font, so if you are already used to the font size in the vscode terminal, it doesn't change. If you want the same size as in the Ubuntu gnome-terminal window, use size 18.

See #35901 for more details.
